### DISCUSSION TITLE
Include Console text in Show/Hide Button

### DISCUSSION THREAD

**Discussion by crashzk** [seDirector/App#44] [Ideas]
It would be interesting if the **Show/Hide** option had written console together, for example:

**Show/Hide Console**

![image](https://user-images.githubusercontent.com/32937653/161781537-a0b89dd5-55f2-4d19-8d86-3ab64f94fde6.png)

_Until a while ago I swore that this button was to show and hide the program window itself._ 😆

**Comment by asheroto** [comment]
Thanks for your feedback. Ideally we want to put the console inside of seDirector rather than it being separate. In progress!
